conduct/code-of-conduct.md

```diff
@@ -44,34 +45,34 @@ except as necessary to protect vulnerable people from intentional abuse.
 
 ### Harassment in online channels
 
-Participants at `<EVENT>` may also partication in online chats, discussion forums, and mailing lists ( including resources such as Slack and Twitter ). Please use these guidelines where engaging with particpants:
+Participants at `<EVENT>` may also participate in online chats, discussion forums, and mailing lists (including platforms such as Slack and Twitter). Please use these guidelines where engaging with particpants:
 
 * Avoid using overtly sexual or offensive usernames or profile photos which might detract from a friendly, safe and welcoming environment for all.
 * Do not publish text/screenshots of anything shared in a private communication channels without explicit consent from the author.  This includes screenshots of private messages to public channels, as well as conversations on public channels to anywhere outside of `<EVENT>` Slack.
 
 ## What To Do If You Witness or Are Subject To Unacceptable Behavior
 
-If you are being harassed, notice that someone else is being harassed, or have any other concerns relating to harassment, please contact a member of event staff immediately. Event staff can be identified by t-shirts/staff badges onsite; and an organizer can be found at the event registration counter at any time. You can contact the `<RESPONSE TEAM>` via `<CONTACT INFO>`.
+If you are being harassed, notice that someone else is being harassed, or have any other concerns relating to harassment, please contact a member of event staff immediately. Event staff can be identified by t-shirts/staff badges onsite; and an organizer can be found at the event registration counter at any time. You can contact the `<RESPONSE TEAM>` at `<EMAIL ADDRESS>`.
```

Comment:
Do you think it's important to have a phone contact too?

Reply:
I think it's optional, I changed from`<CONTACT INFO>` to `<EMAIL ADDRESS>` to emphasize that having a reporting email is essential and "contact info" didn't seem precise enough.  We could change to `<EMAIL ADDRESS and/or PHONE>`?

Reply:
Agreed that email is more important, I think phone can be optional. "and/or" makes the email seem optional, how about `<EMAIL ADDRESS (+ optional PHONE)>`?

Reply:
If you don't think phone is important, we don't need. I just live in a world where emails can take up to a week to be read, but that's clearly not standard.

Reply:
@karawoo Changed to `<EMAIL ADDRESS (+ optional PHONE)>`.